### PR TITLE
up2-leds: adding new basic sample for UP2 built-in LEDs

### PR DESCRIPTION
**Pull request by Propanu**
Signed-off-by: Mihai Tudor Panu <mihai.tudor.panu@intel.com>

### CODE REVIEW

File: up2-leds/maven/src/main/java/iotdk/example/Up2Leds.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    final static int minBrightness = 0;
+    final static int maxBrightness = 255;
+
+    public static void main(String argv[]) throws InterruptedException {
+
+        // Perform a basic platform and version check
+        if (mraa.getPlatformType() != Platform.INTEL_UP2) {
+            System.out.println("This example is meant for the UP Squared board.");
+            System.out.println("Running it on different platforms will likely require code changes!");
+        }
```
Comment (anjgola) [inline up2-leds/maven/src/main/java/iotdk/example/Up2Leds.java]: Does it need a check for the grove pi shield?

File: up2-leds/sample.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "UP Squared LEDs",
```
Comment (anjgola) [inline up2-leds/sample.json]: I think the name should differentiate it from Onboard Blink.

File: up2-leds/maven/src/main/java/iotdk/example/Up2Leds.java (hunk 1 above)
Comment (Propanu) [reply up2-leds/maven/src/main/java/iotdk/example/Up2Leds.java]: No, it uses the built-in LEDs, no shield required.

File: up2-leds/sample.json (hunk 2 above)
Comment (Propanu) [reply up2-leds/sample.json]: Would "UP Squared built-in LEDs" be more clear?

Comment (anjgola) [reply up2-leds/sample.json]: Yes

**Comment by Propanu** [comment]
Pushed title changes as discussed. I've also made it clear that no additional hardware is required for this sample.

**Pull request merged**